projects/js-packages/jetpack-cli/composer.json

```diff
@@ -0,0 +1,40 @@
+{
+	"name": "automattic/jetpack-cli",
+	"description": "Development tools for the Jetpack monorepo",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {},
+	"require-dev": {
+		"automattic/jetpack-changelogger": "@dev"
+	},
+	"autoload": {
+		"classmap": [
+			"src/"
+		]
+	},
+	"scripts": {
+		"phpunit": [
+			"echo 'no phpunit yet'"
+		],
+		"test-coverage": [
+			"echo 'no php'"
+		],
+		"test-php": [
+			"echo 'no php'"
+		],
```

Comment:
If there's no PHP and never will be, just delete this so it won't waste time trying to run it.

Reply:
Thanks, resolving via (to be pushed) 3d9bc67632